templates/thank-you.html

```diff
@@ -1,40 +1,60 @@
 {% extends 'base_layout.html' %}
 {% block title %}MicroK8s - Contact-us{% endblock %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o/edit{% endblock meta_copydoc %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1zAmRxMVFQhUPv9jBjVHJPnWfJMjiB78dQfY__h4pE7o/edit{% endblock
+meta_copydoc %}
 
 {% block content %}
 
-<section class="p-strip is-deep">
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <h1>Thank you for contacting us about MicroK8s</h1>
-      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-    </div>
-    <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
-      <img src="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg" alt="" width="200" height="200">
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col"></div>
+      <div class="col">
+        <h1 class="u-no-margin--bottom">Thank you for contacting us about MicroK8s</h1>
+        <p class="p-heading--2">A member of our team will be in touch<br class="u-hide--small" /> within one working
+          day.
+        </p>
+      </div>
     </div>
   </div>
-</section>
 
-<section class="p-strip--light">
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Ubuntu Advantage</h3>
-      <p>Purchase our desktop support and access Ubuntu experts whenever you need.</p>
-      <p><a  href="https://buy.ubuntu.com">Buy Ubuntu Advantage</a></p>
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col"></div>
+      <div class="col">
+        <hr class="p-rule" />
+        <h2>Learn more about our enterprise services.</h2>
+      </div>
     </div>
+  </div>
 
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Ubuntu OpenStack</h3>
-      <p>Learn about Canonical&rsquo;s industry leading, private cloud infrastructure platform based on Openstack.</p>
-      <p><a href="/openstack">Learn about Ubuntu OpenStack&nbsp;&rsaquo;</a></p>
-    </div>
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col"></div>
+      <div class="col">
+        <hr class="p-rule" />
+        <div class="row">
+          <div class="col-3 col-medium-2">
+            <h5>Ubuntu Pro</h5>
```

Comment:
```suggestion
            <h3 class="p-heading--5>Ubuntu Pro</h3>
```
These should all be semantic h3s

Reply:
but why?

Reply:
We should avoid skipping heading ranks where possible. Source: https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

Reply:
so you mean since in the previous section we are using h2, next section should not utilize h5 (which is 3 levels lower) but h3 instead?

Reply:
correct